package.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
The lock file seems to be missing. It's useful to have all the versions pinned for more consistent builds.

Reply:
I agree; but I was hopping android would be in first to rebase on top of it given that package.json is going to be the common merge conflict.